docs/homologation/requirements.md

```
@@ -7,16 +7,47 @@ title: Requisitos Obrigatórios
+
+   - Não realizar GETs contínuos de alguma entidade para verificar se houve alterações, quando o correto seria executar um Webhook.
+
+   - Modificar entidades sem necessidade (caso de uso: aplicativo que modifica constantemente o estoque dos produtos para 0 e logo em seguida altera para outro valor).
+
+1.2. **Buscar oportunidades de melhoria no fluxo de trabalho dos parceiros**:
+
+   - Construir um aplicativo mais robusto e com menos falhas.
+   - Melhorar a eficiência do seu aplicativo.
+   - Detectar possíveis problemas de segurança.
+
+   ### Exemplo de fluxo de instalação de um aplicativo:
```

markdownlint-cli2 (0.20.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)